docs/userguide/scheduling/resource-propagating.md

```diff
@@ -260,20 +260,81 @@ PropagationPolicy can be configured as follows:
 apiVersion: policy.karmada.io/v1alpha1
 kind: PropagationPolicy
 metadata:
-  name: nginx-propagation
+ name: nginx-propagation
 spec:
+ #...
+ placement:
+  clusterAffinity:
+   exclude:
+    - member1
+    - member3
+  #...
+```
+
+## Deploy deployment into multi set of target clusters
```

Comment:
```suggestion
## Multiple cluster affinity groups
```
The `clusterAffinities` is not used to deploy a resource to `multiple` clusters at the same time.

Reply:
Updated.